sru/__init__.py

```diff
@@ -4,8 +4,5 @@
 sru provides a PyTorch implementation of the simple recurrent neural network cell described
 in "Simple Recurrent Units for Highly Parallelizable Recurrence."
 """
-from .version import __version__
-try:
-    from .sru_functional import *
-except:
-    from sru_functional import *
+from sru.version import __version__  # noqa: F401
+from sru.modules import SRU, SRUCell  # noqa: F401
```

Comment:
looks like you *might* be able to avoid noqa, if you put after this:

```
__all__ = ['SRU', 'SRUCell']
```

https://stackoverflow.com/a/31079085